### PR TITLE
Only install the latest compatible version of timescaledb

### PR DESCRIPTION
**Pull request by idanovinda**
During the installation of timescaledb, in each Postgres version it will create shared objects from all minor versions which support each version. We want to keep the latest compatible version and delete all the previous minor versions.

### CODE REVIEW

File: postgres-appliance/build_scripts/base.sh [hunk 1]
```diff
@@ -124,6 +124,15 @@ for version in $DEB_PG_SUPPORTED_VERSIONS; do
         "postgresql-${version}-pg-stat-kcache" \
         "${EXTRAS[@]}"
 
+    # Clean up timescaledb versions except the highest compatible version
+    highest_version=$(find /usr/lib/postgresql/"${version}"/lib/timescaledb-2.*.so | sort -V | tail -n 1)
+    find /usr/lib/postgresql/"${version}"/lib/ -name 'timescaledb-2.*.so' ! -path "${highest_version}" -exec rm -f {} \;
```
Comment (hughcapet) [inline postgres-appliance/build_scripts/base.sh:129]: https://github.com/zalando/spilo/blob/9edde4d2929d25b27fb50663493ddb056854f4fc/postgres-appliance/build_scripts/compress_build.sh#L24

File: postgres-appliance/build_scripts/base.sh [hunk 2]
```diff
@@ -124,6 +124,19 @@ for version in $DEB_PG_SUPPORTED_VERSIONS; do
...
+    # Clean up timescaledb versions except the highest compatible version
+    exclude_patterns=()
+    exclude_patterns_tsl=()
+    for ts_version in ${TIMESCALEDB}; do
+        exclude_patterns+=(! -name timescaledb-"${ts_version}".so)
+        exclude_patterns_tsl+=(! -name timescaledb-tsl-"${ts_version}".so)
+    done
+    find /usr/lib/postgresql/"${version}"/lib/ -name 'timescaledb-2.*.so' "${exclude_patterns[@]}" -delete;
+
+    if [ "${TIMESCALEDB_APACHE_ONLY}" = "false" ]; then
```
Comment (hughcapet) [inline postgres-appliance/build_scripts/base.sh:136]:
```suggestion
    if [ "${TIMESCALEDB_APACHE_ONLY}" != "true" ]; then
```

**Comment by hughcapet** [comment]
👍

**Comment by idanovinda** [comment]
👍

**Pull request merged**